### ISSUE TITLE
Issues installing on Mojave

### ISSUE THREAD

**Issue by vaccinemedia** [dgsga/netatalk#5]
When running the meson build command I'm getting:
meson.build:1:0: ERROR: Value "c2x" (of type "string") for combo option "C language standard to use" is not one of the choices. Possible choices are (as string): "none", "c89", "c99", "c11", "c17", "gnu89", "gnu99", "gnu11", "gnu17".

I'm assuming it's because I'm running on a non Apple device but please let me know if this is not the case and there is an actual bug here.

**Comment by vaccinemedia** [comment]
It turns out that the xcode command line tools were not the min version required. I assumed that having installed xcode from the app store that it would be the latest / required version. So I had to remove it:
sudo rm -rf /Library/Developer/CommandLineTools

And then Install the Command Line Tools for Xcode 11.3.1 from:
https://developer.apple.com/download/all/

Then I ran "brew install berkeley-db dgsga/netatalk-dbus/dbus-glib docbook-xsl libevent libgcrypt meson mysql libressl pkg-config" again which on my poor i5 Hackintosh took over three hours ;D

And now running "meson setup build" works!
